content/_authors/anthony.bonnemain.md

```diff
@@ -0,0 +1,11 @@
+---
+fullname: Anthony BONNEMAIN
+role: Bizdev
+missions:
+  - start: '2019-05-01'
+    end:
```

Comment:
Il manque un date de fin, tu es fonctionnaire ? sinon il faut mettre fin de cdd ou le temps prévu pour la mission

Reply:
Slt, je suis salarié Pôle Emploi et en mission pour la start up, je ne connais pas précisément la date de fin de celle-ci...

Reply:
Ok, il faudra bien nous notifier dans ce cas à ton départ pour notre gestion.

Bienvenue ;)

Reply:
Ok c'est noté
Merci :)